indico/htdocs/sass/modules/abstracts/_abstract_page.scss

```diff
@@ -206,6 +198,15 @@
 .abstract-review-badges {
     flex-grow: 1;
     color: $gray;
+
+    .js-ratings-toggle {
```

Comment:
The way I always used/saw them is this:

- If there's a class that is unique and clear enough when used in JS I use that one. `toggle-something` for example.
- If there's no such class I add a `js-*` one.
- I never use "generic" style classes (like `icon-delete`) for JS bindings, as nobody would expect technical side-effects when changing/removing an icon